audiophiler/templates/base.html

```diff
@@ -44,6 +44,32 @@
                             My Files
                         </a>
                     </li>
+                    {% if is_rtp or is_eboard %}
+                        <li class="dropdown">
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+                                <span class="glyphicon glyphicon-music"></span> Harold
+                                <span class="caret"></span>
+                            </a>
+                            <ul class="dropdown-menu">
+                                <li>
+                                    <a href="/tour_page">
+                                        <span class="glyphicon glyphicon-user"></span> Tour Mode
+                                    </a>
+                                </li>
+                                <li>
+                                    <a href="/selected">
+                                        <span class="glyphicon glyphicon-tasks"></span> Selected Harolds
+                                    </a>
+                                </li>
+                            </ul>
+                        </li>
+                    {% else %}
+                        <li>
+                            <a href="/selected">
+                                Selected Harolds
+                            </a>
```

Comment:
You set icons for the admin version, but not the non admin version. Is this intended? I'd think if you set it for one you'd set it for both.

Reply:
Intended, doesn’t look good other way around. Also it follows the syntax of conditional